### DISCUSSION TITLE
Бриф

### DISCUSSION THREAD

**Discussion by Darolga** [atls/kino-reporter#5] [General]
**Цели бизнеса и роль сайта**

1. Какое место в вашем бизнесе отведено сайту?
2. Какие приоритетные задачи должен решать?
3. Какие финансовые выгоды вы ожидаете получить?
4. Как сайт может повлиять на ваш бизнес в целом?
5. Какие видите перспективы развития сайта в вашем бизнесе?
6. На чем зарабатывает сайт?
7. Как сайт влияет/ должен повлиять/ на ваш основной вид деятельности?
8. Опишите проблемы существующего сайта, которые требуют решения.

**Пользователь**

1. Кто ваша целевая аудитория? Возраст, пол, социальное положение, интересы.
Если это несколько сегментов, опишите каждый и с какой цели приходит на сайт.
   а)
   б)
   в)

7. Опишите идеальный сценарий поведения пользователя (с точки зрения целей бизнеса)
8. Проводит ли ваша компания исследование пользовталей?
9. Как осуществляется поддержка (взаимодействие, обратная связь) с пользователем?
10. Как осуществляется поддержка сайта? Админы, авторы..
11. Ведется ли поддержка соцсетей? Как часто и какие материалы выкладываются (хотели бы выкладывать)
12. Как хотите упростить жизнь себе и пользователям?

**Бренд**

1. О бренде - лого, фирстиль. Каким должен быть? 
2. а) Примеры (рефы) тех, кто нравится
   б) Примеры тех, кто не нравится.

3. Какие цели и задачи должен решить новый бренд и новый сайт?
4. Есть ли узнаваемость бренда сейчас?
5. Есть ли маркетинговый план? На какой уровень хотите выйти и каких целей достичь?
6. Какие проблемы стоят перед вашим бизнесом, стратегия развития, как бренд должен этому помочь?

**Конкуренты**

1. Кого вы видите основными конкурентами всего бизнеса?
2. Есть ли у конкурентов сайт и какую роль он выполняет?
3. На кого вы ориентируетесь как пример?
4. Покажите решения конкурентов  вам нравятся именно вам и почему?
5. Чем вы лучше других? (Преимущества, фичи, уникальные предложения.)
7. Какие барьеры? Что мешает, тормозит развитие? 
8. Как хотите масштабироваться?

**Comment by AlexeyGovorukhin** [comment]
Цели бизнеса и роль сайта

**СМИ, цель сайта – привлечение аудитории, расширение узнаваемости бренда, повышение экспертности бренда**

Какое место в вашем бизнесе отведено сайту?

**Основной продукт – печатная версия журнала, но в последнее время значимость сайта существенно выросла, сейчас стоит задача повысить посещаемость и цитируемость материалов сайта**

Какие приоритетные задачи должен решать? 

**Максимальный охват релевантной аудитории, KPI > 30К уников в день**

Какие финансовые выгоды вы ожидаете получить?

**500K руб. в месяц**

Как сайт может повлиять на ваш бизнес в целом?

**Активное развитие посещаемости сайта, и повышение узнаваемости бренда позволит существенно повысить рекламные доходы издания (журнал + сайт + соцсети)**

Какие видите перспективы развития сайта в вашем бизнесе?

**Создание экспериментальной креативной площадки с новыми форматами контента, создание каталога (базы данных) фильмов и персон, собственный медиаплеер**

На чем зарабатывает сайт?

**Вложения инвесторов, реклама**

Как сайт влияет/ должен повлиять/ на ваш основной вид деятельности?

**Несущественно/ стать основным источником привлечения аудитории и рекламодателей / медиапортал**

Опишите проблемы существующего сайта, которые требуют решения.

**Падение трафика из рекомендательных систем (Дзен, Пульс), требуется существенное повышение поискового трафика и улучшение поведенческих**

Пользователь


Кто ваша целевая аудитория? Возраст, пол, социальное положение, интересы. Если это несколько сегментов, опишите каждый и с какой цели приходит на сайт. 

**а) м 40% /ж 60% 

б) 24-45 

в) подборки, рекомендации, рецензии**


Опишите идеальный сценарий поведения пользователя (с точки зрения целей бизнеса)

**Регулярность посещений (чаще 1 раза в неделю), глубина 2+, время на сайте 3 мин +**

Проводит ли ваша компания исследование пользователей?

**Гугл аналитика, Яндекс метрика**


Как осуществляется поддержка (взаимодействие, обратная связь) с пользователем?

**Социальные сети**

Как осуществляется поддержка сайта? Админы, авторы..

**Редакция, админы**

Ведется ли поддержка соцсетей? Как часто и какие материалы выкладываются (хотели бы выкладывать)

**Ведется, материалы выкладываются через RSS (Твиттер, ВК, ФБ), ручная публикация (Телеграм, ИГ)**

Как хотите упростить жизнь себе и пользователям?

**Ускорение работы сайта, оптимизация поиска**

Бренд

О бренде - лого, фирстиль. Каким должен быть?

а) Примеры (рефы) тех, кто нравится б) Примеры тех, кто не нравится.

**УЖЕ ЕСТЬ**

Какие цели и задачи должен решить новый бренд и новый сайт?

**БРЕНД ПРЕЖНИЙ
Сайт – рост аудитории**

Есть ли узнаваемость бренда сейчас?

**Да**

Есть ли маркетинговый план? На какой уровень хотите выйти и каких целей достичь?

**См выше**


Какие проблемы стоят перед вашим бизнесом, стратегия развития, как бренд должен этому помочь?

**Узкая ниша, высокая конкуренция**

Конкуренты
Кого вы видите основными конкурентами всего бизнеса?

**Kinopoisk.ru
Kino-teatr.ru
Kinoart.ru
Kinomania.ru
Film.ru
Filmz.ru
GQ.ru
OK.ru**

Есть ли у конкурентов сайт и какую роль он выполняет?

**СМИ**

На кого вы ориентируетесь как пример?

**Hollywoodreporter.com**

Покажите решения конкурентов вам нравятся именно вам и почему?

Hdrezka.org
 
https://yadi.sk/i/U_Sv_PLO5UALbQ

https://yadi.sk/i/vKVtSxfSSUfwzA


**Всплывающие окна (карточки, сниппеты) по наведению или клику на объект (картинка, ссылка)**

Чем вы лучше других? (Преимущества, фичи, уникальные предложения.)

**Оригинальный (авторский) контент, интервью, новости**

Какие барьеры? Что мешает, тормозит развитие?

**Узкая ниша, высокая конкуренция**

Как хотите масштабироваться?

**Расширять аудиторию за счёт смежных ниш – лайфстайл, мода, сплетни, техника и технологии, игры**

**Comment by AlexeyGovorukhin** [comment]
UPD
70% посетителей - на мобильных устройствах
60% посетителей - из поиска + 20% прямые зазходы + 10%  из  рекомендательных + 10% остальное
90% заходят на страницу материала (а не на главную)

**Comment by TorinAsakura** [comment]
@AlexeyGovorukhin https://www.figma.com/proto/pUWwcuHo61RiwLEzNztoGZ/Web-site?page-id=0%3A1&node-id=16%3A487&viewport=241%2C48%2C1.02&scaling=min-zoom&starting-point-node-id=16%3A487

**Reply by AlexeyGovorukhin** [reply]
> @AlexeyGovorukhin https://www.figma.com/proto/pUWwcuHo61RiwLEzNztoGZ/Web-site?page-id=0%3A1&node-id=16%3A487&viewport=241%2C48%2C1.02&scaling=min-zoom&starting-point-node-id=16%3A487

Коллеги, большое спасибо за подробный разбор.

Постараюсь максимально развернуто прокомментировать.

Корень зла вот в чём: сайт делали за 2 недели своими силами (холдинговые разрабы), основная задача была максимальное совместимость с thr.ru, чтобы максимально оперативно перенести структуру и админку. Дальше начались костыли WP. Собственно, отсюда и строка поиска, и оформление материалов на главной и прочие косяки, которые исправить имеющимися инструментами и ресурсами оказалось невозможно.


НЕДОСТАТКИ

Структура меню у нас изначально сложная была - есть две структуры: 1) формат (рецензия, статья, интервью) и 2) рубрика (кино, сериалы, стиль, театр). Эти структуры пересекаются, то есть может быть кино-новости, сериалы-новости, театр-премии и т.п., иногда материал попадает сразу в несколько рубрик. Как распутать пока не понимаем.
Подкасты появились через 2 года после создания сайта и была задача сделать плеер максимально видимым.

горизонтальные отбивки (раздел "трейлеры") на главной делали по предложению предыдущего шеф-редактора, реализация по началу работала нормально, потом трейлеры стали не актуальны - главная в целом приносила минимум трафика, трейлеры тем более.

Задача автоподгрузки страниц была увеличить глубину просмотров, реализована не лучшим образом, но настроить у разрабов не получилось, 

Мобильная адаптация также многожды дорабатывалась, но избавится от больших разрывов между карточками материалов не получилось

Со шрифтом хотелось бы иметь возможность менять его размер либо сделать покрупнее.

Группировка статей на странице - время визуального взаимодействия пользователя с контентом минимальное, в основном считывают картинку и заголовок, если сразу не зацепило - пользователь уходит, поэтому решили предлагать больше вариантов.

ПРЕДЛОЖЕНИЯ 

Реализация базы данных фильмов/сериалов/персон стоит в плане, скорее всего летом сделаем основной блок

Календарь премьер фильмов и сериалов, а также график выхода отдельных серий - да, тоже планируем, совмещается с базой данных

Подписку на рассылку делали ранее через mailchimp, малоэффективно

Взаимодействие с контентом - да, ЛК - не уверен, что нужно, сразу встаёт вопрос о статусе оператора персональных данных, проще дать возможность авторизироваться через соцсети, почту и т.п.

Про второй уровень навигации не очень понятно. Теги и рубрики есть уже .

ЗАДАЧИ

1) Кратное увеличение трафика (цель - 30К уников в день). После того, как Дзен существенно ограничил переходы, трафик рухнул. Сейчас ещё отвалились Гугл новости. Полностью оптимизировать сайт под требования Google Page Speed тоже пока не удалось.
2) Мобильный трафик да, больше и будет расти, очевидно. Нужно оптимизировать сайт под мобильные просмотры.
3) Интерес пользователей - очень конкурентная сущность. Про кино, телевидение и селебов сейчас пишут все кому не лень. У нас есть определенные ограничения на хайповость и неоднозначность заголовков. Из приведённых примеров, разве что про порно хайповый, но такие новости раз в год можно выдавать, чтобы не скатиться совсем в желтизну.
 
ЦЕЛИ ПОЛЬЗОВАТЕЛЯ И ПУТИ ИХ ДОСТИЖЕНИЯ

Нужно активнее использовать инструменты гугл, в том числе выдачу в сниппетах, карточках, ответах и т.п.

![2022-04-04_18-25-06](https://user-images.githubusercontent.com/87372347/161582939-e2e76a48-31c8-4ee0-8245-82a95cd74380.png)
_пример_

Оптимизация сайта под требования гугла
Оптимизация фидов под новостные и рекомендательные сервисы

ЗАДАЧИ ДИЗАЙНА

Создать интуитивно понятный, визуально привлекательный ресурс. Концепция mobile-first - да, начало разработки с внутренних страниц - да! 80% трафика идёт на внутренние страницы, внутренние переходы чаще не через главную, а по внутренним ссылкам, через прокрутку и т.п. 

КОНКУРЕНТЫ

Кинопоиск больше БД и VOD сервис
Кино-театр - отчасти, на этом ресурсе вложено много сил и средств на продвижение.
Искусство кино - отчасти, на очень узкую аудиторию
Киномания, фильм.ру, Фильмз.ру - да
THe Hollywood Reporter - отчасти, ранее у нас была франшиза, хорошо знаем. У них много инсайдов, аналитики, расследований. При этом очень долго сайт был с убогим дизайном, что не мешало быть влиятельным.


ВЗАИМОДЕЙСТВИЕ ПОЛЬЗОВАТЕЛЯ С САЙТОМ

Цели сервиса

Увеличить вовлечённость и охват - да!
Привязка пользователя к контенту - да, но нужно понять какими методами, скорее пуши, чем почтовая рассылка
Продажа журнала - отчасти, количество продаваемых журналов ограничено
Продажа рекламных мест на сайте

СТРУКТУРА САЙТА

Основной точкой входа будет страница статьи/фильма/персоны
